src/Microsoft.TemplateEngine.Edge/Settings/Scanner.cs

```diff
@@ -277,21 +303,11 @@ private bool TryCopyContentForNonFileSystemBasedMountPoints(IMountPoint mountPoi
             }
             catch (IOException)
             {
+                _environmentSettings.Host.LogDiagnosticMessage($"Error copying scanLocation: {scanLocation} into the content dir: {path}", "Install");
                 diskPath = null;
                 return false;
             }
 
-            try
```

Comment:
Are these getting cleaned up somewhere else that we're able to get rid of the delete here?

Reply:
Yes, near the end of Scan(), at line 55. 

TryCopyIntoContentForNonFileSystemBasedMountPoints() gets called from ScanForComponents() if any .dll's exist in the scan source. When the call occurs, there is no guarantee that any of those .dll's contain components. When there are no components, we still need to scan source in the packages directory for doing a template / langpack scan. 

Both scanning for components, and scanning for templates / langpacks will set source.AnythingFound = true when something is found. That flag is used at the end of Scan() to decide whether to do this delete.

Reply:
Looking further, I found a problem - if a sources has components, it never gets deleted from Packages/ ands end up both in Packages/ and Content/, but should only be in Content/. Will fix.